_targets_eval_postprocessing.R

```diff
@@ -1207,6 +1227,13 @@ hub_comparison_plots <- list(
     command = c(
       "COVIDhub-4_week_ensemble",
       "UMass-trends_ensemble",
+      "CEPH-Rtrend_covid",
+      "MOBS-GLEAM_COVID",
+      "CMU-TimeSeries",
+      "SGroup-RandomForest",
+      "UMass-gbq",
+      "UMass-sarix",
+      "UT-Osiris",
       "cfa-wwrenewal(real-time)",
       "cfa-wwrenewal(retro)",
       "cfa-hosponlyrenewal(retro)"
```

Comment:
Long run we will presumably want this to be generated programmatically, no? Fine for now.

Reply:
Yeah I think we can set this up to be specified in the config. Here I want to generate a figure to see all the models and then choose which ones we want to visualize so I will update this at some point and move to config after looking at results. Think we need to justify the choice of the one or two non ensemble models (i.e. best performing)

Reply:
Good point, but I meant more that this could be all of `covidhub_models_to_score` in the first instance